examples/stable/test/property/ic_api/json/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
```

Comment:
This tsconfig is totally wrong, just follow what the other tests do in the ic_api directory.